lib/plugins/aws/invoke/index.js

```diff
@@ -28,6 +28,16 @@ class AwsInvoke {
     // validate function exists in service
     this.options.functionObj = this.serverless.service.getFunction(this.options.function);
 
+    // if data is provided via CLI, sanitize it
+    if (typeof this.options.data === 'string') {
+      try {
+        this.options.data = JSON.parse(this.options.data);
```

Comment:
Nice catch @dzimine 👍 

Maybe we should open up an issue so that we can track that.
Could you do that @dzimine? Thanks!

Related to https://github.com/serverless/serverless/issues/3691.